Source/SimCacheModule/Source/Subsystems/AircraftTracker/AircraftTracker.cpp

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2024 Steven Frost and Orion Lyau
+
+#include "Subsystems/AircraftTracker/AircraftTracker.h"
+
+#include "Events/AircraftPositionUpdatedEvent.h"
+
+#include <utility>
+
+// -----------------------------------------------------------------------------
+
+DEFINE_LOG_CATEGORY( AircraftTracker, Info )
+
+// -----------------------------------------------------------------------------
+
+namespace Subsystems
+{
+
+// -----------------------------------------------------------------------------
+
+AircraftTracker::AircraftTracker(
+	Utils::NativeEventDispatcher& InternalEventDispatcher,
+	SimConnect::ISimConnectClient& SimConnectClient
+)
+	: InternalEventDispatcher( InternalEventDispatcher )
+	, SimConnectClient( SimConnectClient )
+	, UserAircraftPositionUpdatedHandle()
+{
+}
+
+// -----------------------------------------------------------------------------
+
+bool AircraftTracker::Initialize()
+{
+	// TODO: Only bind the position updated event when we have a tracked cache
```

Comment:
Regarding this todo note -- are we ever going to want to track the position of the user aircraft / ownship outside having a tracked cache?  (e.g. maybe on the cache selection frontend / FS2024 EFB app, we might want to be able to get the current location to filter the list by some form of proximity, so it doesn't suggest caches on the other side of the globe?)